charts/rancher-windows-gmsa/3.0.0/app-readme.md

```diff
@@ -1,9 +1,28 @@
 # Windows GMSA Admission Webhook
 
-This chart creates the GMSA CRD, Credential, and Admission Webhook. The official documentation and tutorials can be found [here](https://github.com/kubernetes-sigs/windows-gmsa).
+This chart creates the `GMSACredentialSpec` CRD, a default `GMSACredentialSpec` to be used in workloads, and Admission Webhook. The official documentation and tutorials can be found [here](https://github.com/kubernetes-sigs/windows-gmsa).
 
 ## Prerequisites
 
 - Active Directory that supports Group Managed Service Accounts
 - A Group Managed Service Account
 - Kubernetes v1.21+
+
+## Required RBAC Changes  
+
+In order to create workloads running under custom service accounts, an additional `ClusterRoleBinding` must be created to bind each desired service account with the `rancher-windows-gmsa` role created by this chart.
+
+```yaml 
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: <CLUSTER_ROLE_BINDING_NAME>
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: rancher-windows-gmsa
+subjects:
+  - kind: ServiceAccount
+    name: <SERVICE_ACCOUNT_NAME>
+    namespace: <SERVICE_ACCOUNT_NAMESPACE>
+```
```

Comment:
Couple of notes here.

1. We shouldn't specify that you need to create a **Cluster**RoleBinding. You should only need a Role + RoleBinding per namespace for the best security posture here
2. can we move this to the README.md? app-README.md should be smaller.